### DISCUSSION TITLE
browsers migration to brew in macOS

### DISCUSSION THREAD

**Discussion by AWHubGit** [Homebrew/discussions#28] [Everyday usage]
I have macOS 10.15.7 with Homebrew 2.5.2.
On macOS are installed browsers Firefox and Google Chrome. Installed by manually downloading DMG-/PKG-files and installing them.

What to do to have these browsers manged by brew? How to migrate installed browsers to brew? Is uninstallation of traditonally installed browsers first needed? And then installation same browsers with brew cask? Or can these browser installs be attached to brew? How?

**Comment by SMillerDev** [comment]
No, you can't attach existing installs through brew. I believe `brew cask install --force google-chrome` will cleanly override the existing chrome though
